cog/cog-client.cpp

```diff
@@ -55,6 +56,7 @@ class GTaskAsyncContext : public AsyncCallerContext {
 typedef struct
 {
   CognitoIdentityProviderClient internal;
+  gchar *region;
```

Comment:
Style nitpick: we are trying to use plain C types (`char`, `int`, etc.) in new code.

Reply:
I've defined the property as a enum so this is outdated now.